src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -293,6 +293,7 @@ class PopoverReportActionContextMenu extends React.Component {
                         reportID={this.state.reportID}
                         reportAction={this.state.reportAction}
                         draftMessage={this.state.reportActionDraftMessage}
+                        selection={this.state.selection}
```

Comment:
Same comment here which I used for the description key. 

>  How the whole menu description goes to each item's description? Can you think of a way to do this with ContextMenuActions?

We don't need this selection at all.